pkg/webhook/v1/webhook.go

```diff
@@ -0,0 +1,27 @@
+package v1
+
+import (
+	"k8s.io/client-go/kubernetes"
+	v1 "open-cluster-management.io/api/work/v1"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+type ManifestWorkWebhook struct {
+	kubeClient kubernetes.Interface
```

Comment:
I do not thinks it is needed in this webhook, right?

Reply:
It is used here: https://github.com/open-cluster-management-io/work/pull/164/files#diff-c05eb54e7b9381bee7827595cf0c481f6c38395bdafaccfb692136bfddba67aaR158